proxy/src/managers/default/buffer_manager.rs

```diff
@@ -191,6 +200,24 @@ impl<T: BufferManagerType> BufferManager<T> {
         )
         .await
     }
+
+    pub fn block_user(&mut self, user_account_id: AccountId, blocked_account_id: AccountId) {
+        self.block_list
+            .add_to_block_list(user_account_id, blocked_account_id);
```

Comment:
you already know youre working on a block list so why not just name the function add or block?